// @enocean-js/eep-transcoder/src/eep/a5-3f-7f-eltako-shutters.js
export const a53f7feltakoshutters = {
  number: '0x7F',
  title: 'Universal Eltako Shutters',
  status: 'released',
  description: 'Eltako Shutter blinds from https://www.eltako.com/fileadmin/downloads/en/_main_catalogue/Gesamt-Katalog_ChT_gb_lowRes.pdf',
  case: [{
    datafield: [{
      data: 'LRN Bit',
      shortcut: 'LRNB',
      description: 'LRN Bit',
      bitoffs: '28',
      bitsize: '1',
      enum: {
        item: [{
          value: '0',
          description: 'Teach-in telegram'
        }, {
          value: '1',
          description: 'Data telegram'
        }]
      }
    }, {
      data: 'Runtime',
      shortcut: 'RT',
      description: 'Runtime in 100ms or in seconds depending on DBO_Bit1',
      info: {},
      bitoffs: '0',
      bitsize: '16',
      enum: {
        item: [{
          min: '0',
          max: '65535',
          scale: {
            min: '0',
            max: '6553.5'
          },
          unit: 's'
        }, {
          value: '0',
          description: 'Do not run'
        }]
      }
    },{
      data: 'Command',
      shortcut: 'CMD',
      description: 'Command',
      info: {},
      bitoffs: '16',
      bitsize: '8',
      enum: {
        item: [{
          value: '0',
          description: 'Stop',
          info: {}
        }, {
          value: '1',
          description: 'Go up'
        }, {
          value: '2',
          description: 'Go down'
        }]
      }
    },{
      data: 'Lock Bit',
      shortcut: 'LOCK',
      description: 'Lock/unlock the actuator for pushbutton',
      bitoffs: '29',
      bitsize: '1',
      enum: {
        item: [{
          value: '0',
          description: 'unlock'
        }, {
          value: '1',
          description: 'lock'
        }]
      }
    },{
      data: 'Runtime Unit Switch',
      shortcut: 'RTU',
      description: 'Specify runtime units',
      bitoffs: '30',
      bitsize: '1',
      enum: {
        item: [{
          value: '0',
          description: 'Seconds'
        }, {
          value: '1',
          description: '100ms'
        }]
      }
    },{
      data: 'TeachIn Activation',
      shortcut: 'TIA',
      description: 'Teach-in mode activation 3x within 2s = delete ID',
      bitoffs: '26',
      bitsize: '1',
      enum: {
        item: [{
          value: '0',
          description: 'nothing'
        }, {
          value: '1',
          description: 'Activate Teach In'
        }]
      }
    }]
  }],
  originalIndex: 138,
  eep: 'a5-3f-7f',
  rorg_title: '4BS Telegram',
  rorg_number: '0xA5',
  func_title: 'Universal',
  func_number: '0x3F',
  submitter: [
    'ELTAKO'
  ]
}
